pipeline/inputs/tail.md

```diff
@@ -19,7 +19,7 @@ The plugin supports the following configuration parameters:
 | Read\_from\_Head | For new discovered files on start \(without a database offset/position\), read the content from the head of the file, not tail.                                                                                                                                                                                                                                                                                                                                                                          | False |
 | Refresh\_Interval | The interval of refreshing the list of watched files in seconds.                                                                                                                                                                                                                                                                                                                                                                                                                                         | 60 |
 | Rotate\_Wait | Specify the number of extra time in seconds to monitor a file once is rotated in case some pending data is flushed.                                                                                                                                                                                                                                                                                                                                                                                      | 5 |
-| Ignore\_Older | Ignores files which modification date is older than this time in seconds. Supports m,h,d \(minutes, hours, days\) syntax.                                                                                                                                                                                                                                  |  |
+| Ignore\_Older | Ignores files which modification date is older than this time in seconds. Supports m,h,d \(minutes, hours, days\) syntax. Configuring this option can improve performance when your Path matches a lot of older files that are no longer being actively modified.                                                                                                                                                                                                                                |  |
```

Comment:
Are we sure that this resolves the issue yet? Customer compliant about fluent bit is 1 failure per 2 weeks.
Maybe we should wait for confirmation before publishing these doc updates?

Reply:
Well this doc update doesn't mention any issue, it just says it improves performance, which is true right? Monitoring fewer files means you have less work to do. 

We are not yet certain of the importance of this config for one specific customer. I think the doc update is not specific to that.

Reply:
Now that we know that really you need both log rotation and this, and log rotation is more important, I will close this PR